docs/references/backend/sessions/authenticate-request.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: authneticateRequest() | User Object
+description: Networkless method to verify a token passed from the frontend.
+---
+
+# `authenicateRequest()`
+
+Networkless method to authenicate a token passed from the frontend..
+
+```tsx
+const authStatus = await clerkClient.authenticateRequest();
+```
+
+## `AuthenticateRequestParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `request` | `IncomingMessage` | `req` object |
+| `secretKey?` | `string` | Secret Key (same as apiKey) |
+| `publishableKey?` | `string` | Clerk Publishable Key |
+| `domain?` | `string` | Domain for the application |
+| `isSatellite?` | `boolean` | Is this a satellite domain in a multiple domain setup? |
```

Comment:
in the next props you are referring to `multi-domain setup` instead of `multiple domain setup`. I would suggest we keep the same wording for consistency.